src/System.Security.Cryptography.Csp/tests/DSACryptoServiceProviderTests.cs

```diff
@@ -187,6 +194,7 @@ public static void NonExportable_Ephemeral()
         }
 
         [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)] // No support for CspParameters on Unix
```

Comment:
Please add the negative tests.  All the stuff that we want to throw should throw.